### PR TITLE
Added infrastructure for sending notifications. Also added YouHaveBee…

### PR DESCRIPTION
**Pull request by krishna-patel-ch**
__Short description outlining key changes/additions__

Added infrastructure for sending notifications. Also added YouHaveBeenAddedToAcsp notification.

__JIRA Ticket Number__

IDVA6-1457

### Type of change

* [ ] Bug fix
* [x] New feature
* [ ] Breaking change
* [x] Infrastructure change

### Pull request checklist

* [x] I have added unit tests for new code that I have added
* [x] I have added/updated functional tests where appropriate
* [x] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,6 +30,8 @@
         <google-http-client.version>1.44.2</google-http-client.version>
         <api-helper-java.version>3.0.1</api-helper-java.version>
         <flapdoodle.version>4.16.1</flapdoodle.version>
+        <email-producer-java.version>3.0.2</email-producer-java.version>
+        <version.kafka-clients>2.6.0</version.kafka-clients>
```
Comment (tferns-ch) [inline pom.xml:34]: Might be better to have .version at the end to be consistent with the others?

```suggestion
        <kafka-clients.version>2.6.0</kafka-clients.version>
```

**Review by tferns-ch — APPROVED** [review]
nice work, just a small comment

**Review by tferns-ch — APPROVED** [review]
nice work, just a small comment

File: pom.xml (hunk 1 above)
Comment (krishna-patel-ch) [reply pom.xml]: Good spot! Yeah, I agree. I've made the change

**Pull request merged**